mmcv/ops/csrc/pytorch/mlu/mlubind.cpp

```diff
@@ -0,0 +1,2 @@
+#include "pytorch_cpp_helper.hpp"
```

Comment:
We have a `cudabind.cpp` to reduce the nvcc compile time(It take long time to compile template for multiple cuda arch).
If there is no such issue for compiler of mlu, we can remove this file and register implementation in each ops. Like what we did for cpu implementation.

Reply:
Okay, this PR doesn't include any operators and no need of this file.

Maybe I should add mlubind in the future with MLU operators at same time?

Reply:
Ok

Reply:
Done, mlubind.cpp is removed.